src/ansys/fluent/core/solver/flobject.py

```diff
@@ -437,6 +437,9 @@ def _while_executing_command(self):
         """Avoid additional processing while executing a command."""
         return nullcontext()
 
+    def __eq__(self, other):
+        return self.flproxy == other.flproxy and self.path == other.path
```

Comment:
Object identity no longer holds between built setting object and corresponding conventional setting object as the address of the built setting object is now different although it is transformed into a conventional setting object.